src/style.js

```diff
@@ -21,15 +21,17 @@ export default class extends Component {
   }
 }
 
-export function flush() {
-  const ret = {}
-
-  for (const {props} of components) {
-    ret[props.styleId] = props.css
+function componentMap() {
+  const ret = new Map()
+  for (const c of components) {
+    ret.set(c.props.styleId, c)
```

Comment:
Since we're returning a Map now it's not going to be compatible regardless. Also I thought it would be best if the 2 outputs from this module returned the same data format.